storey/drivers.py

```diff
@@ -223,7 +224,9 @@ def _build_feature_store_update_expression(self, aggregation_element, additional
             for name, value in additional_data.items():
                 if name.casefold() in self.saved_engine_words.keys():
                     name = f'`{name}`'
-                expressions.append(f'{name}={self._convert_python_obj_to_expression_value(value)}')
+                expression_value = self._convert_python_obj_to_expression_value(value)
+                if expression_value:
```

Comment:
If the timestamp is null and we don't delete the attribute, we're preserving whatever value already exists in the database, which doesn't seem like correct behaviour.

Reply:
yes, we talked about it yesterday :)

Reply:
And now we won't forget :)